content/en/docs/setup/install/istioctl/index.md

```diff
@@ -650,7 +650,15 @@ The --set flag also creates any intermediate nodes in the path that are missing
 
 ## Uninstall Istio
 
-To uninstall Istio, run the following command:
+To uninstall Istio completely, run the following command:
+
+{{< text bash >}}
+$ istioctl x uninstall <your original installation options> --purge
+{{< /text >}}
+
+The optional `--purge` flag is used to indicate removing all Istio resources, including cluster scoped resources shared between different Istio control planes.
```

Comment:
this should warn that the CRs that are  removed will remove all configuration data for the Isito control plane.

Reply:
Do you mean putting this in a warning block or the description is not clear enough?

Reply:
Added as a warning block

Reply:
i think steve might have meant saying something like, "including cluster scoped resources that may still be used by another Istio version, and Istio user configuration. Use only if you are sure you want to completely remove every Istio related resource from the cluster."